README.md

```diff
@@ -49,6 +49,9 @@ SVGinOT Font demos (Firefox only):
 The font can be installed for a user or system-wide. Get the latest version
 from releases: https://github.com/eosrei/twemoji-color-font/releases
 
+Some distributions like Fedora and openSUSE already ship the font.
+Check https://pkgs.org/download/twemoji to see which do.
```

Comment:
This link 404s.

Reply:
Weird, it worked before I made the PR. I'll look for an alternative…

Reply:
Agreed, it's weird, the site's search tool takes me to that URL, but the URL doesn't work by itself...

Reply:
Use instead:

- https://pkgs.org/download/twemoji-color-font (for openSUSE)
- https://pkgs.org/download/twemoji-color-font-ttf (for FreeBSD / NetBSD)

Regards

-